model/histogram/float_histogram.go

```diff
@@ -94,8 +94,8 @@ func (h *FloatHistogram) CopyToSchema(targetSchema int32) *FloatHistogram {
 		Sum:           h.Sum,
 	}
 
-	c.PositiveSpans, c.PositiveBuckets = mergeToSchema(h.PositiveSpans, h.PositiveBuckets, h.Schema, targetSchema)
-	c.NegativeSpans, c.NegativeBuckets = mergeToSchema(h.NegativeSpans, h.NegativeBuckets, h.Schema, targetSchema)
+	c.PositiveSpans, c.PositiveBuckets = shrink(h.PositiveSpans, h.PositiveBuckets, h.Schema, targetSchema, false)
+	c.NegativeSpans, c.NegativeBuckets = shrink(h.NegativeSpans, h.NegativeBuckets, h.Schema, targetSchema, false)
```

Comment:
Wouldn't this `shrink` function be a drop-in replacement for `mergeToSchema` in other call sites as well (such as `Add` and `Sub` methods)? In that case it would benefit immediately from the decent test coverage that those methods already have.

Reply:
Yes, you are right. This commit is only for affirm my idea is align with @krajorama, is not the version ready for review, since krajorama also stands for this idea, I will remove the `mergeToSchema` method.